rules-reviewed/eap6/jboss-eap5/jboss-eap5-xml.windup.xml

```diff
@@ -191,25 +191,33 @@
                 </classification>
             </perform>
         </rule>
-        <rule id="jboss-eap5-xml-10000">
-            <when>
-                <xmlfile matches="//*[local-name()='classloading']" in="jboss-classloading.xml"/>
+        <rule id="jboss-eap5-7-xml-10000">
+           <when>
+                <xmlfile matches="/jbc:classloading">
+                    <namespace prefix="jbc" uri="urn:jboss:classloading:1.0"/>
+                </xmlfile>
             </when>
             <perform>
-                <hint title="JBoss 5 classloader configuration (jboss-classloading.xml)" effort="5" category-id="optional">
+                <classification issue-display-mode="detail-only" title="JBoss 5 classloader configuration (jboss-classloading.xml)" effort="0" category-id="mandatory">
+                    <description>The `jboss-classloading.xml` file allows customization of classloading in JBoss EAP 5.</description>
+                    <link title="JBoss EAP 5 Class Loading and Modules" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/5/html/Microcontainer_User_Guide/sect-JBoss_Microcontainer_User_Guide-The_ClassLoading_Layer-ClassLoading.html"/>
+                    <link title="JBoss EAP 7 Class Loading and Modules" href="https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/development_guide/#class_loading_and_modules"/>
+                    <tag>classloading</tag>
+                </classification>
+                <hint title="JBoss 5 classloader configuration (jboss-classloading.xml)" effort="5" category-id="mandatory">
                     <message>
                         In JBoss EAP 6, the classloading configuration is not done through `jboss-classloading.xml`.
                         Since EAP 6 it is of modular nature, based on JBoss Modules.
                         Learn how to divide your application's dependencies into modules
                         and what dependencies and APIs are available automatically.
                     </message>
-                    <link title="JBoss EAP 6 Class Loading and Modules" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html/Development_Guide/chap-Class_Loading_and_Modules.html"/>
-                    <link title="JBoss EAP 6 Class Loading and Modules" href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/5/html/Microcontainer_User_Guide/sect-JBoss_Microcontainer_User_Guide-The_ClassLoading_Layer-ClassLoading.html"/>
+                    <link title="JBoss EAP 7 Class Loading and Modules" href="https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/development_guide/#class_loading_and_modules"/>
```

Comment:
Maybe it's worth keeping the links to the JBoss EAP 6 documentation here and also the in the above classification to be consistent w/ the target?

Reply:
Yes, that is correct. I will fix it.

Reply:
Fixed.